src/components/Detail/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Col, Image, Row } from 'react-bootstrap'
+
+const Detail = ({ data }) => {
+  const item = data.results[0]
```

Comment:
It'd be good to add an appropriate name for the const (like comics or movie) or just use destructuring.